bodhi/bugs.py

```diff
@@ -64,8 +71,13 @@ def getbug(self, bug_id):
 
     def comment(self, bug_id, comment):
         try:
+            if len(comment) > 65535:
+                raise InvalidComment("Comment is too long: %s" % comment)
             bug = self.bz.getbug(bug_id)
             bug.addcomment(comment)
+        except InvalidComment as err:
+            log.exception("Invalid comment for bug #%d" % bug_id)
+            log.exception(err)
```

Comment:
So, ``log.exception`` already automatically logs the exception *in addition* to whatever string you pass it.  SO, this second ``log.exception(err)`` statement would log the exception twice (three times in total).

Maybe get rid of line 80 and replace line 79 with:

```python
log.exception("Comment too long for bug #%d:  %s" % (bug_id, comment))
```

That way we can see what the comment is so we can reason about it.